src/Entities/Boat.cpp

```diff
@@ -39,6 +40,35 @@ void cBoat::SpawnOn(cClientHandle & a_ClientHandle)
 
 
 
+void cBoat::BroadcastMovementUpdate(const cClientHandle * a_Exclude)
```

Comment:
I just noticed there's a [vehicle move packet](https://wiki.vg/Protocol#Vehicle_Move_.28clientbound.29). Should we be using that instead?

Also, could we exclude the rider from the broadcasts? I assume the rider's client knows the boat's position anyway.

Reply:
The clientbound vehicle move packet is implemented. Ideally we would implement the serverbound one too, but I had some issues getting it to play nicely with the client.

I tried to exclude the rider, but once you leave the boat, the client sends the boat back to the spawn position until you rejoin.

Reply:
That's a shame, oh well.

Reply:
For future reference: serverbound vehicle move packet is implemented, clientbound is not. I mixed them up.